gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/components/api-analytics-empty/api-analytics-empty.component.html

```diff
@@ -0,0 +1,40 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card class="empty-card">
```

Comment:
We have an empty state card in ui-particles. I don't know it we should add a state that allow you to have this design

( see https://particles.gravitee.io/?path=/story/components-card-empty-state--card-empty-state )